pkg/metrics/policyfiltermetrics/policyfiltermetrics.go

```diff
@@ -53,15 +53,26 @@ var (
 	}, []string{"subsys", "op"})
 )
 
+var (
+	PolicyFilterHookContainerNameMissingMetrics = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Since this metric doesn't have any labels you can use `NewCounter` instead of `NewCounterVec`. Then you can also delete L74 initializing this metric (it will get initialized automatically).

Reply:
Also please run `make metrics-docs` when adding a new metric to have it autodocumented. Thanks!

Reply:
Thank you for the review! Rookie mistakes, sorry :sweat_smile: 
Initially, I only saw your first message; therefore, I didn't run `make metrics-docs` before pushing the changes but now I pushed the changes to docs too, thank you (I was afraid I would need to document them manually:) )!

Reply:
Can you rephrase the commit message to have a shorter subject? It's a detail, I know, but otherwise CI fails with:
```
Error: ERROR:CUSTOM: Please avoid long commit subjects (max: 75, found: 78)
```
https://github.com/cilium/tetragon/actions/runs/8553067127/job/23447481238?pr=2231

Reply:
sure, I agree that long commit messages are hard to read, I removed the word `metric` from the subject as it should be (hopefully) self-explanatory from `policyfiltermetrics`